dashboard/app/models/unplugged.rb

```diff
@@ -24,4 +24,32 @@
 #
 
 class Unplugged < Level
+  def self.create_from_level_builder(params, level_params)
+    create!(level_params.merge(user: params[:user], game: Game.unplugged, level_num: 'custom'))
+  end
+
+  def assign_attributes(new_attributes)
+    i18n_strings = {
+      'title' => new_attributes.delete(:title),
+      'desc' => new_attributes.delete(:description),
+    }
+    update_i18n(new_attributes[:name], i18n_strings)
+
+    super(new_attributes)
+  end
+
+  def title
+    I18n.t("data.unplugged.#{name}.title")
+  end
+
+  def description
+    I18n.t("data.unplugged.#{name}.desc")
+  end
+
+  def update_i18n(name, new_strings)
+    unplugged_yml = File.expand_path('config/locales/unplugged.en.yml')
+    i18n = File.exist?(unplugged_yml) ? YAML.load_file(unplugged_yml) : {}
+    i18n.deep_merge!({'en' => {'data' => {'unplugged' => {name => new_strings}}}})
+    File.write(unplugged_yml, i18n.to_yaml(line_width: -1))
+  end
```

Comment:
This is an interesting pattern- looks like this provides a form-field entry for a couple properties (`title` and `description`) that will be written not to the `.level` file, but directly into the English i18n file so that they will be (eventually) localized across languages through our existing i18n pipeline.

If this pattern works well in practice, I'd love to see it eventually standardized/extracted into a helper module of some sort, since it seems like a pattern we could leverage anywhere we need to create levels containing localized content (and I'd love to settle on a single process for localized content in levels, rather than conflicting processes unique to each individual level type).